esx_service/vmdk_ops_test.py

```diff
@@ -1399,6 +1408,22 @@ def testPolicyUsageCount(self):
 
         vsan_policy.delete(self.name)
 
+
+def setUpModule():
+    # Let's make sure we are testing a local DB
+    os.system(ADMIN_RM_LOCAL_AUTH_DB)
```

Comment:
I am little bit confused here. What is the "local DB" name here for test? Is seems we don't pass the local DB name to the command, right?